1.2/spec.md

```diff
@@ -627,10 +627,34 @@ Issue: [ocr_carea vs ocrx_block](https://github.com/kba/hocr-spec/issues/28)
 
 ### `ocrx_line`
 
-Issue: [ocr_line vs ocrx_line](https://github.com/kba/hocr-spec/issues/19)
-
-  * any kind of "line" returned by an OCR system that differs from the standard ocr_line above
+  * any kind of "line" returned by an OCR system that differs from [[#ocr_line]]
   * might be some kind of "logical" line
+  * examples include line continuations and rowspan in tables
+
+<div class="example">
+
+Consider the following snippet, containing a wide-spaced heading broken over
+two physical lines:
+
+<figure>
+  <img
+    width=600
+    alt="Wide spaced two line heading"
+    src="../images/akf-widespaced-heading.png"/>
+</figure>
+
+An OCR engine could produce the following output, indicating the two physical
+lines that form a single logical line:
+
+```html
+...
+<span class="ocrx_line">
```

Comment:
ocrx_line is engine-specific line markup. It exists for those cases where your OCR engine outputs text lines that don't correspond to "normal" text lines.

The most common case is if you apply an engine that's not capable of column segmentation to a multi-column document and you want to prevent subsequent processing stages from assuming that the text lines it gets contain text in reading order.

Basically, if you use ocrx_line instead of ocr_line, you're (intentionally) breaking most subsequent processing, since most OCR output processing will look for ocr_line tags (and assume they are in reading order).